### PR TITLE
feature-44 cleaning up Main2.java

### PR DESCRIPTION
**Pull request by KareemZafan**

### CODE REVIEW

File: src/Main2.java [hunk 1]
```diff
@@ -4,10 +4,7 @@ public class Main2{
...
  public static void main(String[] args){
  for(int i = 0 ; i < 10; ++i)
-   System.out.println("Configuration Management Second Day"); 
-System.out.println("Configuration Management Second Day1"); 
-System.out.println("Configuration Management Second Day2"); 
-System.out.println("Configuration Management Second Day3"); 
+   System.out.println("Configuration Management Second Day"+(i+1)); 
   }
 
 }
```
Comment (AbdallahALBanna) [inline src/Main2.java:12]: comment by banna

File: src/Main2.java [hunk 2]
```diff
@@ -4,10 +4,7 @@ public class Main2{
 
  public static void main(String[] args){
  for(int i = 0 ; i < 10; ++i)
-   System.out.println("Configuration Management Second Day"); 
-System.out.println("Configuration Management Second Day1");
```
Comment (Husseinsobieh) [inline src/Main2.java:8]: duplicate change or fired

**Review by AbdallahALBanna — CHANGES_REQUESTED** [review]

**Review by Husseinsobieh — CHANGES_REQUESTED** [review]

**Review by MaryamMagdy570 — CHANGES_REQUESTED** [review]
modify number of days

File: src/Main2.java [hunk 3]
```diff
@@ -4,10 +4,7 @@ public class Main2{
 
  public static void main(String[] args){
  for(int i = 0 ; i < 10; ++i)
-   System.out.println("Configuration Management Second Day"); 
-System.out.println("Configuration Management Second Day1"); 
-System.out.println("Configuration Management Second Day2"); 
-System.out.println("Configuration Management Second Day3");
```
Comment (AhmedOsama96) [inline src/Main2.java:10]: remove duplicates

**Review by AhmedOsama96 — APPROVED** [review]

**Review by Ali-Abdelhakem-00 — APPROVED** [review]

**Review by BasantMohamed33 — APPROVED** [review]

File: src/Main2.java (hunk 2 above)
Comment (KareemZafan) [reply src/Main2.java:8]: it is already updated

**Review by Husseinsobieh — APPROVED** [review]

**Review by MaryamMagdy570 — APPROVED** [review]

**Review by AbdallahALBanna — APPROVED** [review]

File: src/Main2.java [hunk 4]
```diff
@@ -3,8 +3,10 @@
 public class Main2{
 
  public static void main(String[] args){
- for(int i = 0 ; i < 10; ++i)
-   System.out.println("Configuration Management Second Day"+(i+1)); 
+ for(int i = 0 ; i < 3; ++i)
```
Comment (Husseinsobieh) [inline src/Main2.java:6]: better than before

**Review by Husseinsobieh — APPROVED** [review]

**Pull request merged**